config/crd/bases/kinda.rocks_dbusers.yaml

```diff
@@ -62,6 +62,23 @@ spec:
               secretName:
                 description: SecretName name that should be used to save user's credentials
                 type: string
+              templates:
+                description: Templates to add custom entries to ConfigMaps and Secrets
+                items:
+                  description: Tempaltes to add custom entries to comfigmaps and secrets
```

Comment:
typo
- Tempaltes
- comfigmaps

and the non camel case is intended?

Reply:
Not intended, I'll fix

Reply:
Actually, templates shouldn't be added to dbusers at all, I just forgot to remove it

Reply:
Remove templates from dbusers